### PR TITLE
Improve zipcode counting notebook by adding GPU backed WKT parser

### PR DESCRIPTION
**Pull request by isVoid**
## Description

This PR improves the zipcode counting notebook by utilizing cudf string and list column methods to parse WKT as geometry column. This achieves 40X speed up comparing to parsing on host.

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cuspatial/blob/HEAD/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

File: notebooks/ZipCodes_Stops_PiP_cuSpatial.ipynb [hunk 1]
```diff
@@ -135,33 +135,114 @@
...
+    "# d_zip = cuspatial.from_geopandas(h_zip)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6fdaedfc-2d7f-4d73-a9b8-e3a8131bea2f",
+   "metadata": {},
+   "source": [
+    "Instead, we can use cudf list and string method to parse the wkt into coordinates and build a geoseries.\n",
+    "Without roundtripping to host, cudf provides ~40x speed up by computing on GPU. \n",
```
Comment (harrism) [inline notebooks/ZipCodes_Stops_PiP_cuSpatial.ipynb:178]: This is awesome! e.g. Might need to state details of this comparison. e.g. which GPU and CPU.

**Review by harrism — APPROVED** [review]
Speedup is awesome. Suggesting one bit of explanation.

File: notebooks/ZipCodes_Stops_PiP_cuSpatial.ipynb (hunk 1 above)
Comment (isVoid) [reply notebooks/ZipCodes_Stops_PiP_cuSpatial.ipynb:178]: I remember this is mentioned below:

```
Reference machine: Intel(R) Xeon(R) CPU E5-2698 v4 @ 2.20GHz v.s. NVIDIA Tesla V100 SXM2 32GB
```

**Review by jarmak-nv — APPROVED** [review]
Love it!

On my personal machine:
- Old method: 23.6s
- This method: 1.9s

:tada:

**Pull request merged**